okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyConnection.java

```diff
@@ -147,14 +147,14 @@ private SpdyConnection(Builder builder) throws IOException {
       okHttpSettings.set(Settings.INITIAL_WINDOW_SIZE, 0, OKHTTP_CLIENT_WINDOW_SIZE);
     }
 
-    hostName = builder.hostName;
+    service = protocol + ":" + builder.hostName + ":" + builder.socket.getPort();
```

Comment:
standard to use a colon between protocol and host? `http/1.1:twitter.com:443`  ? I'd expect different formatting; perhaps `twitter.com:443 (http/1.1)` ?